extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -261,7 +261,7 @@ export class DraggableDrawer extends AMP.BaseElement {
       ? this.startListeningForTouchEvents_()
       : this.stopListeningForTouchEvents_();
 
-    this.headerEl_.toggleAttribute('desktop', !isMobile);
+    this.headerEl_?.toggleAttribute('desktop', !isMobile);
```

Comment:
What happens if the `headerEl_` is built after the `onUIStateUpdate` is sent? The attribute would be missing, right?

Reply:
Since `onUIStateUpdate` is initialized in `layoutCallBack`, and `headerEl_ ` is built in `buildCallback` I'm not sure how this could ever not be built after the `onUIStateUpdate` is sent.

By following the stack trace this looks like the component should be built.
Is there ever a context or render mode where `layoutCallBack` would be called before `buildCallBack`?